### PR TITLE
feat: add player mode to slider

### PR DESCRIPTION
**Pull request by jesperwiner**
adjust arrow on dropdown bubble

add icon

### CODE REVIEW

File: src/Molecules/Slider/Slider.tsx [hunk 1]
```diff
@@ -187,6 +221,10 @@ const Slider: Component = ({
     }
   };
 
+  const getstyle = {
```
Comment (stigdanielek) [inline src/Molecules/Slider/Slider.tsx:224]: Maybe consider another name for this object. Either `getStyle` (but then it should probably be a function) or just `style`?

File: src/Molecules/Slider/Slider.tsx [hunk 2]
```diff
@@ -84,16 +95,19 @@ const Slider: Component = ({
...
   const isControlled = isNumber(controlledValue);
+  const [hoverPosition, setHoverPosition] = useState<number | null>(null);
+  const [hoverVisible, setHoverVisible] = useState<boolean>(false);
   const [valueInternal, setValueInternal] = useState(defaultValue || min);
   const value = isControlled ? controlledValue! : valueInternal;
   const trackPercent = valueToPercent(value, min, max);
-  const handlePosition = `calc(${trackPercent}% - ${
-    variant === 'big' ? `${THUMB_BIG_PX}` : `${THUMB_SMALL_PX}`
-  }px * ${trackPercent * 0.01})`;
+  const handlePosition = `calc(${trackPercent}% - ${getKnobSize(variant) / 2}px * ${1})`;
```
Comment (stigdanielek) [inline src/Molecules/Slider/Slider.tsx:110]: Can't we remove the `* ${1}` here or am I missing something?

File: src/Molecules/Slider/Slider.tsx (hunk 1 above)
Comment (jesperwiner) [reply src/Molecules/Slider/Slider.tsx:224]: removed it and made inline again as it was before

Comment (jesperwiner) [reply src/Molecules/Slider/Slider.tsx:224]: had it outside for a while and then not

File: src/Molecules/Slider/Slider.tsx (hunk 2 above)
Comment (jesperwiner) [reply src/Molecules/Slider/Slider.tsx:110]: used to be a 0,01 multiplier there and I forgot the clean up all the way

**Review by stigdanielek — DISMISSED** [review]

**Review by mic1000 — APPROVED** [review]

**Review by kk-11 — APPROVED** [review]

**Review by jonnakollin — DISMISSED** [review]

**Review by alvelid — DISMISSED** [review]

**Review by kk-11 — APPROVED** [review]

**Review by LifeofMichal — APPROVED** [review]

**Pull request merged**